HowTos/gateway.rst

```diff
@@ -227,6 +226,13 @@ If it is left blank and there is already a load balancer in the specified VPC, i
 
 You can set the ELB name if there is no existing ELB in the specified VPC.
 
+VPN Protocol
```

Comment:
ELB Protocol.

Reply:
Sherry wants to name it VPN protocol. I think it makes sense